maquetado/scss/partials/_styles.scss

```diff
@@ -1,22 +1,32 @@
 @import 'fonts';
+@import 'fixed-sizes';
 
 .container {
   font-family: $roboto-stack;
+  padding: 50px;
+}
+
+.content-wrapper {
   height: 100%;
   overflow-y: auto;
 }
 
 .main-wrapper {
   display: flex;
-  flex-flow: column nowrap;
   height: 100vh;
+  flex-flow: column nowrap;
 }
 
 .tech-wrapper {
-  padding: 10px;
-  display: flex;
-  flex-flow: row wrap;
-  justify-content: flex-start;
-  margin: 0 auto 60px auto;
-  width: 90%;
+  display: grid;
+  grid-template-columns: 2fr 1fr 2fr;
```

Comment:
I'd recommend setting the same size to all the columns since visually they look alike. Also, take a look at the [`repeat()` function](https://developer.mozilla.org/en-US/docs/Web/CSS/repeat) it does a couple of interesting things!